protos/dcs/world/world.proto

```diff
@@ -14,7 +14,7 @@ service WorldService {
 }
 
 message GetAirbasesRequest {
-  optional dcs.common.Coalition coalition = 1;
+  dcs.common.Coalition coalition = 1; // Use COALITION_UNSPECIFIED to get airbases for all coalitions
```

Comment:
I wonder whether we should just call it `COALITION_ANY` in the case of the coalition, since it is actively used as a value to _request all_?

Reply:
Lemme see if we can do that and tell the linter to ignore the fact that it doesn't end in `_UNSPECIFIED`

Reply:
I think `protolint` is probably what we should standardise on in our CI in which case I can make this change using a comment to disable the rule for the line.

Looking at how this is used I think `COALITION_ALL` makes more sense than `COALITION_ANY`. What do you think?

Reply:
`COALITION_ALL` sounds good 👍 
There are two comments about `COALITIONS_UNSPECIFIED` left, LGTM otherwise

Reply:
Whoops, fixed and remove other references to `unspecified`